src/plugins/textToSpeech/KoreAWSPolly/kore-aws-polly.ts

```diff
@@ -0,0 +1,153 @@
+import BaseTTS from "../BaseTTS";
+import * as AWS from 'aws-sdk';
+declare const document: any;
+declare const window: any;
+class speakTextWithAWSPolly extends BaseTTS {
+    name = 'speakTextWithAWSPollyClass';
+    config = {
+        'AWS.config.region': 'ap-south-1', // Region
```

Comment:
@UdayAp-Kore 
Remove the confidential keys immediately every where

Reply:
> @UdayAp-Kore Remove the confidential keys immediately every where

@RajasekharBa-Kore  It was there before also.
Anyhow deleted now.